### PR TITLE
Synced block components in api reference page

### PR DESCRIPTION
**Pull request by terrikwak**
## Proposed changes

- Describe your changes to communicate to the maintainers why we should accept this pull request.
- If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Minor Issues and Typos
- [ ] Major Content Contribution
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to reach out. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [ ] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn-docs/blob/master/CONTRIBUTING.md)
- [ ] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn-docs)
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large content contribution, kick off the discussion by explaining why you would suggest the content contribution, etc...

### CODE REVIEW

File: docs/klaytn/design/computation/execution-model.md [hunk 1]
```diff
@@ -48,17 +48,34 @@ A block is a crucial element of the Klaytn blockchain because the blockchain lit
...
 | Component | Description |
 | :--- | :--- |
+| baseFeePerGas | QUANTITY | The base fee per gas. This value is returned only when EthTxTypeCompatibleBlock is activated for that block number. |
+| blockScore | QUANTITY | Former difficulty. Always 1 in the BFT consensus engine |
+| currentBlock  | QUANTITY | The current block, same as `klay_blockNumber`.               |
+| extraData | DATA | The "extra data" field of this block. |
+| gasUsed | QUANTITY | The total used gas by all transactions in this block. |
+| governanceData | DATA | RLP encoded governance configuration |
+| highestBlock  | QUANTITY | The estimated highest block.                                 |
+| knownStates  | QUANTITY | The number of known state entries that still need to be pulled.  If the sync mode is not "fast", zero is returned. |
```
[the remark this answers is not in the document]
Comment (terrikwak) [reply docs/klaytn/design/computation/execution-model.md:58]: @aidan-kwon should i delete??

File: docs/klaytn/design/computation/execution-model.md [hunk 2]
```diff
@@ -48,17 +48,34 @@ A block is a crucial element of the Klaytn blockchain because the blockchain lit
...
-| GasUsed | The number of gas used to process transactions in the block. |
-| Time | An integer value equal to the Unix timestamp when the block was generated. |
-| Extra | RLP encoded string which includes validators list, proposer's seal and committed validators' seals |
-| Transactions | The transactions included in the block. |
+| proposer | 20-byte DATA | The address of the block proposer. |
+| receiptsRoot | 32-byte DATA | The root of the receipts trie of the block. |
+| reward | 20-byte DATA | The address receiving block reward. |
+| size | QUANTITY | Integer the size of this block in bytes. |
+| startingBlock | QUANTITY | The block at which the import started (will only be reset, after the sync reached his head). |
+| pulledStates  | QUANTITY | The number of state entries processed until now.  If the sync mode is not "fast", zero is returned. |
```
[the remark this answers is not in the document]
Comment (terrikwak) [reply docs/klaytn/design/computation/execution-model.md:67]: @aidan-kwon  it was from here https://docs.klaytn.com/dapp/json-rpc/api-references/klay/block#klay_syncing

File: docs/klaytn/design/computation/execution-model.md [hunk 3]
```diff
@@ -48,17 +48,29 @@ A block is a crucial element of the Klaytn blockchain because the blockchain lit
 
 | Component | Description |
```
Comment (aidan-kwon) [inline docs/klaytn/design/computation/execution-model.md:49]: 위쪽에 State 의 Component는 대문자로 시작하는데, 지금 변경으로 Block에 Component는 소문자로 시작하네요.
맞추면 좋을거 같아요.

Comment (aidan-kwon) [reply docs/klaytn/design/computation/execution-model.md:49]: 타입도 기존에는 없었기에 제거해도 좋을거 같습니다.

File: docs/klaytn/design/computation/execution-model.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (aidan-kwon) [reply docs/klaytn/design/computation/execution-model.md]: @terrikwak Thanks!

File: docs/klaytn/design/computation/execution-model.md [hunk 4]
```diff
@@ -37,28 +37,40 @@ The table below shows the account data that are stored in the state.
...
 | :--- | :--- |
+| baseFeePerGas | The base fee per gas. This value is returned only when EthTxTypeCompatibleBlock is activated for that block number. |
+| blockScore | Former difficulty. Always 1 in the BFT consensus engine |
+| extraData | The "extra data" field of this block. |
+| gasUsed | The total used gas by all transactions in this block. |
+| governanceData | RLP encoded governance configuration |
+| logsBloom | The bloom filter for the logs of the block. `null` when it is pending block. |
+| number | The block number. `null` when it is pending block. |
 | ParentHash | The hash of the block's parent block. |
```
Comment (terrikwak) [inline docs/klaytn/design/computation/execution-model.md:58]:
```suggestion
| parentHash | The hash of the block's parent block. |
```

**Comment by terrikwak** [comment]
@aidan-kwon  does this look okay now?

File: docs/klaytn/design/computation/execution-model.md [hunk 5]
```diff
@@ -37,28 +37,40 @@ The table below shows the account data that are stored in the state.
...
+| totalBlockScore | Integer of the total blockScore of the chain until this block. |
+| transactionsRoot | The root of the transaction trie of the block. |
+| timestamp | The Unix timestamp for when the block was collated. |
+| timestampFoS | The fraction of a second of the timestamp for when the block was collated. |
+| transactions | Array of transaction objects, or 32-byte transaction hashes depending on the last given parameter. |
+| voteData | RLP encoded governance vote of the proposer |
+
+{% hint style="success" %}
+**NOTE**: The `baseFeePerGas` field is added from Klaytn v1.8.0
+{% endhint %}
```
Comment (aidan-kwon) [inline docs/klaytn/design/computation/execution-model.md:73]: How about removing this? There is a more accurate explanation in `baseFeePerGas` component explanation.

Comment (terrikwak) [reply docs/klaytn/design/computation/execution-model.md:73]: @kjhman21  what do you think?

Comment (kjhman21) [reply docs/klaytn/design/computation/execution-model.md:73]: @aidan-kwon In that case, we also remove this note for consistency. What do you think?
https://github.com/klaytn/klaytn-docs/pull/432/files#diff-4c6417c53149e19723cc8457e48659577079a5e30bfa5bd2880201a8b8290883R125-R127

Comment (aidan-kwon) [reply docs/klaytn/design/computation/execution-model.md:73]: @kjhman21 The link transfers me to this PR.

Comment (kjhman21) [reply docs/klaytn/design/computation/execution-model.md:73]: @aidan-kwon oh, that's true. Actually this same hint is also in the docs/dapp/json-rpc/api-references/klay/block.md. I think we need to remove both if you want to remove this one.

Comment (aidan-kwon) [reply docs/klaytn/design/computation/execution-model.md:73]: I agree with you @kjhman21 . And, I think we better remove all `NOTE` in block.md file. How do you think?

Comment (terrikwak) [reply docs/klaytn/design/computation/execution-model.md]: i removed the notes

**Review by aidan-kwon — APPROVED** [review]
Thank you!

**Pull request merged**